<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferenceDialog</class>
 <widget class="QDialog" name="PreferenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6" columnstretch="1,1">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="horizontalSpacing">
        <number>12</number>
       </property>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBoxLogLevel">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>Debug</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Info</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Warning</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Error</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelLogLevel">
         <property name="text">
          <string>Log Level</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelTheme">
         <property name="text">
          <string>Theme</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelLanguage">
         <property name="text">
          <string>Language</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBoxLanguage">
         <item>
          <property name="text">
           <string>English (US)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>简体中文</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBoxTheme"/>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabStartExit">
      <attribute name="title">
       <string>Start &amp;&amp; Exit</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBoxMainWindow">
         <property name="title">
          <string>Main Window</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,1">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <property name="horizontalSpacing">
           <number>12</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelMainWindowStartState">
            <property name="text">
             <string>Start State</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxMainWindowStartState">
            <property name="currentIndex">
             <number>3</number>
            </property>
            <item>
             <property name="text">
              <string>Last position and size</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Minimize</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Maximize</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Center with default size</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Center with last size</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Center with custom size</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Custom position and size</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
            <item>
             <widget class="QLabel" name="labelLeft">
              <property name="text">
               <string>Left</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditLeft">
              <property name="text">
               <string notr="true">0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
            <item>
             <widget class="QLabel" name="labelTop">
              <property name="text">
               <string>Top</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditTop">
              <property name="text">
               <string notr="true">0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
            <item>
             <widget class="QLabel" name="labelWidth">
              <property name="text">
               <string>Width</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditWidth">
              <property name="text">
               <string notr="true">1280</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
            <item>
             <widget class="QLabel" name="labelHieght">
              <property name="text">
               <string>Height</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditHieght">
              <property name="text">
               <string notr="true">720</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxLog">
         <property name="title">
          <string>Log</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,1">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <property name="horizontalSpacing">
           <number>12</number>
          </property>
          <property name="verticalSpacing">
           <number>6</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBoxSaveExit">
            <property name="text">
             <string>Save before Exit</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLineEdit" name="lineEditLogDir"/>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonLogDir">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxTheme">
         <property name="title">
          <string>Theme</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkBoxRandomTheme">
            <property name="text">
             <string>Start with Random Theme</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFont">
      <attribute name="title">
       <string>Font</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBoxUIFont">
         <property name="title">
          <string>UI Font</string>
         </property>
         <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <property name="horizontalSpacing">
           <number>12</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelFontUI">
            <property name="text">
             <string>Font Family</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QFontComboBox" name="fontComboBoxUI">
            <property name="editable">
             <bool>false</bool>
            </property>
            <property name="currentText">
             <string notr="true">DejaVu Sans</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBoxFontSizeUI">
            <property name="currentText">
             <string notr="true">11</string>
            </property>
            <property name="currentIndex">
             <number>5</number>
            </property>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>11</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>18</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>22</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>24</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>26</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>28</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>36</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>48</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>72</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelFontSizeUI">
            <property name="text">
             <string>Font Size</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelFontStyleUI">
            <property name="text">
             <string>Font Style</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBoxFontStyleUI">
            <item>
             <property name="text">
              <string>Regular</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bold</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Italic</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bold Italic</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxMonoFont">
         <property name="title">
          <string>Monospaced Font</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <property name="horizontalSpacing">
           <number>12</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelFontMono">
            <property name="text">
             <string>Font Family</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QFontComboBox" name="fontComboBoxMono">
            <property name="editable">
             <bool>false</bool>
            </property>
            <property name="currentText">
             <string notr="true">Andale Mono</string>
            </property>
            <property name="fontFilters">
             <set>QFontComboBox::MonospacedFonts</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelFontStyleMono">
            <property name="text">
             <string>Font Style</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBoxFontStyleMono">
            <item>
             <property name="text">
              <string>Regular</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bold</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Italic</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bold Italic</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelFontSizeMono">
            <property name="text">
             <string>Font Size</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBoxFontSizeMono">
            <property name="currentText">
             <string notr="true">11</string>
            </property>
            <property name="currentIndex">
             <number>5</number>
            </property>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>11</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>18</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>22</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>24</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>26</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>28</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>36</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>48</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>72</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
